Package.resolved

```diff
@@ -69,7 +69,7 @@
         "repositoryURL": "https://github.com/tuist/xcodeproj.git",
         "state": {
           "branch": null,
-          "revision": "0f563e2d7d604499e7b57a28c78ff23d5c545acd",
+          "revision": "aefcbf59cea5b0831837ed2f385e6dfd80d82cc9",
```

Comment:
This hash is updated automatically. This change cause no effects.
Because, they seem to be almost same 😄  https://github.com/tuist/XcodeProj/compare/0f563e2d7d604499e7b57a28c78ff23d5c545acd...aefcbf59cea5b0831837ed2f385e6dfd80d82cc9